docs/.vuepress/theme/global-components/Supporters.vue

```diff
@@ -127,6 +127,7 @@ export default {
           img {
             width: 100%;
             height: auto;
+            max-height: 100%;
```

Comment:
Wasn't this a safari fix?

Reply:
I recall a safari fix but this doesnt not affect Safari currently and actually should have been in place since the refactor - it keeps images within the confines of its wrapper so they 1. dont get cut off and 2. dont stretch under certain circumstances.